src/v2/Utils/__tests__/logger.jest.ts

```diff
@@ -8,6 +8,7 @@ describe("logger", () => {
   describe("#createLogger", () => {
     let logger
     beforeEach(() => {
+      // eslint-disable-next-line no-console
```

Comment:
Given that this file is testing logging, I'd say we probably could benefit from just hoisting the disable to the top of the file here.